src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -169,3 +213,49 @@ void ov::npuw::s11n::read(std::istream& stream, std::shared_ptr<ov::Node>& var)
     var->output(0).set_tensor_ptr(tensor_dummy);
     var->set_friendly_name(*names.begin());  // any_name ?
 }
+
+void ov::npuw::s11n::read_any(std::istream& stream, ov::Any& var) {
+    // FIXME: ugly, but cannot use .read(stream) here due to its usage of operator>>()
+    int type_int;
+    read(stream, type_int);
+    AnyType type = static_cast<AnyType>(type_int);
+    if (type == AnyType::STRING) {
+        std::string val;
+        read(stream, val);
+        var = std::move(val);
+    } else if (type == AnyType::CHARS) {
+        std::string val;
+        read(stream, val);
+        var = std::move(val);
```

Comment:
but didn't `CHARS` stand for `const char*` in `Any` when you serialized it?

Reply:
I serialize it as std::string to keep it simple